app/assets/javascripts/surveys/modules/SurveyAdmin.js

```diff
@@ -172,12 +176,18 @@ const SurveyAdmin = {
   },
 
   getQuestion(id) {
-    return $.ajax({
-      url: `/surveys/question_group_question/${id}`,
-      method: 'get',
-      dataType: 'json',
-      contentType: 'application/json',
+    return fetch(`/surveys/question_group_question/${id}`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': SurveyAdmin.$csrf_token.content
+      },
       success: $.proxy(this, 'handleConditionalQuestionSelect')
+    }).then((response) => {
+      if (response.ok) {
+        $.proxy(this, 'handleConditionalQuestionSelect');
```

Comment:
Does this still work even though the initial request isn't using jquery anymore?

Reply:
You're right, this was not working, I pushed a commit that fixes this (also replaced the use of $.proxy here since it's deprecated)